invenio_rdm_records/theme/templates/invenio_rdm_records/record_landing_page.html

```diff
@@ -20,19 +20,20 @@
   <div class="row">
     <div class="col-sm-8 col-md-8 col-left">
       {%- block record_body %}
-      <span class="label label-info" title="Publication date">{{ record.publication_date|to_date|format_date(format='long') }}</span>
+      <span class="label label-info"
+        title="Publication date">{{ record.publication_date|to_date|format_date(format='long') }}</span>
       <span class="label record-version"> | Version {{ record.version }}</span>
       <div class="pull-right">
         <span class="label label-default">{{ record.resource_type | vocabulary_title('resource_type') }}</span>
         {% if record.access_right in ["embargoed", "closed"] %}
         <span class="label label-danger titlecase">
-        {% elif record.access_right == "restricted" %}
-        <span class="label label-warning titlecase">
-        {% else %}
-        <span class="label label-success titlecase">
-        {% endif %}
-        {{ record.access_right }} Access
-        </span>
+          {% elif record.access_right == "restricted" %}
+          <span class="label label-warning titlecase">
+            {% else %}
+            <span class="label label-success titlecase">
+              {% endif %}
```

Comment:
minor: probably an auto-indent happened here...

Reply:
yes, have to take a look at the html formatting. nothing seems to be 100% suitable to be automated :(